### ISSUE TITLE
Mocking properties on sealed class results in Missing mocked calls inside every { ... } block

### ISSUE THREAD

**Issue by cliffred** [mockk/mockk#991]
### Expected Behavior
Mocking properties on sealed class works.

### Current Behavior

Given:
Kotlin: 1.7
Java: 17
MockK: 1.13.3

Running this test:
```kotlin
import io.mockk.every
import io.mockk.mockk
import org.junit.jupiter.api.Test

sealed class Foo {
    abstract val name: String
}

class FooImpl : Foo() {
    override val name = "foo"
}

@Test
fun test() {
    mockk<Foo> {
        every { name } returns "bar"
    }
}
```

Results in:
```
Exception in thread "main" io.mockk.MockKException: Missing mocked calls inside every { ... } block: make sure the object inside the block is a mock
	at io.mockk.impl.recording.states.StubbingState.checkMissingCalls(StubbingState.kt:14)
	at io.mockk.impl.recording.states.StubbingState.recordingDone(StubbingState.kt:8)
	at io.mockk.impl.recording.CommonCallRecorder.done(CommonCallRecorder.kt:47)
	at io.mockk.impl.eval.RecordedBlockEvaluator.record(RecordedBlockEvaluator.kt:64)
	at io.mockk.impl.eval.EveryBlockEvaluator.every(EveryBlockEvaluator.kt:30)
	at io.mockk.MockKDsl.internalEvery(API.kt:94)
	at io.mockk.MockKKt.every(MockK.kt:143)
```

- Using Kotlin 1.6 and / or Java 11 with MockK 1.12.8 runs successfully.
- Changing the sealed class to abstract runs successfully.
- The same exception is thrown on any combination of Kotlin 1.6 / 1.7 and Java 11 / 17 with MockK 1.13.3.

**Comment by m-burst** [comment]
I have already fixed this with #988, however the fix is still unreleased.
@Raibaz do you have any plans to release a new version sometime soon? It is currently blocking us from updating to JVM 17 target

**Comment by cliffred** [comment]
Duplicate of https://github.com/mockk/mockk/issues/938
